docs/mesos/README.md

```diff
@@ -7,56 +7,58 @@
+- **Mesos 0.24:** Unsupported. Calico works as a proof of concept, but is no longer supported.
 
-Have any questions on these tutorials? Is there any material you want to see covered? Let us know on our [Slack channel](https://calicousers-slackin.herokuapp.com/).
+### Support for Adding Calico to an Existing Mesos Cluster
+The following tutorials cover installing Netmodules on a fresh machine which does not yet have Mesos installed.
+This is because Calico requires Netmodules, a Mesos Networking Module, which must be compiled with the source header files of the active Mesos installation in order to ensure compatability. If your source installation files exist on your Mesos Slave, then it is possible to follow steps 1-3 and step 6 of the [Manual Calico-Mesos installation Guide](ManualInstallCalicoMesos.md) to add Netmodules and Calico to your existing cluster.
 
-## 1. Prepare Master and Agent Nodes
-The [Mesos Host Preparation tutorial](PrepareHosts.md) will walk you through hostname and firewall configuration for compatability between Calico and Mesos.
+###  Launching Tasks with Calico + Mesos
+IP-Per-Container Networking with Calico is an opt-in feature for Mesos Frameworks that launch tasks with [Networkinfo](https://github.com/apache/mesos/blob/0.26.0-rc3/include/mesos/mesos.proto#L1383). This means that your favorite Mesos Frameworks will not work with Calico until they have opted to include Networkinfo when launching tasks. Currently, this is limited to Mesos tasks launched via Marathon, with support for more frameworks growing. 
 
-## 2. Prepare Core Services
-Zookeeper and etcd serve as the backend datastores for Mesos and Calico, respectively. The [Core Services Preparation tutorial](PrepareCoreServices.md) will walk you through setting up both services using Docker.
+Since the Mesos Docker Containerizer does not support Module hooks, external networking is incompatible with docker containers in 0.26. Modifications are being made to the Mesos Containerizer to launch docker containers in future versions of Mesos, which will work with Calico out of the box going forward.
 
-## 3. Calico
-The following tutorials will help you install the Calico services required for each agent in your Mesos cluster.
+## 1. Prepare Master Node
+The [Mesos Master Preparation tutorial](MesosMasterPreparation.md) 
+will walk you through firewall configuration for compatability 
+between Calico and Mesos.
 
-### Calico With Docker (Recommended)
-Calico is primarily distributed as a Docker container. Follow one of the tutorials below to set that up.
+Zookeeper and etcd serve as the backend datastores for Mesos and 
+Calico, respectively. This tutorial also provides a quick start 
+guide to walk you through setting up both services using Docker.
 
-a.) [Install the Calico Networking in Mesos from RPM](RpmInstallCalico.md)
+## 2. Install
```

Comment:
Can we put a better title here: 

## 2. Install Mesos Slave, Netmodules, and Calico